lib/ansible/modules/cloud/oneandone/oneandone_user.py

```diff
@@ -0,0 +1,424 @@
+#!/usr/bin/python
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: oneandone_user
+short_description: Configure 1&1 users.
+description:
+     - Create, remove, update a user
+       This module has a dependency on 1and1 >= 1.3.0
+version_added: "2.6"
+options:
+  state:
+    description:
+      - Define a user's state to create, remove, or update.
+    required: false
+    default: 'present'
+    choices: [ "present", "absent", "update" ]
```

Comment:
`update` is not ansible compliant

A task defines a state, so `present` should already update the user according to the params given if needed (and return changed in this case). Please change the logic accordingly or provide some information why `update` is needed.